llvm/test/CodeGen/AArch64/sinksplat.ll

```diff
@@ -1,5 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
-; RUN: llc -mtriple=aarch64-linux-gnu -o - %s | FileCheck %s
+; RUN: llc -mtriple=aarch64-linux-gnu -hoist-const-loads=false -o - %s | FileCheck %s
```

Comment:
THIS IS FOR LINE 12: It doesn't look like the load is critical to the test - I think we care about sinking the splat into the loop so that we can use the right form of smlal (i.e. the indexed variant). So perhaps you can just update the test with the load hoisted out?

Reply:
Do you mean to hoist the load manually in a separate commit or just update the check lines together with the patch?

Reply:
Sorry, what I said was confusing! I mean the latter - just recreate the CHECK lines with the patch without adding `-hoist-const-loads=false`. So long as the remaining instructions in the loop look the same as before I think it should be valid.